src/command_modules/azure-cli-eventgrid/setup.py

```diff
@@ -54,7 +54,10 @@
         'azure',
         'azure.cli',
         'azure.cli.command_modules',
-        'azure.cli.command_modules.eventgrid'
+        'azure.cli.command_modules.eventgrid',
+        'azure.cli.command_modules.eventgrid.sdk',
+        'azure.cli.command_modules.eventgrid.sdk.models',
+        'azure.cli.command_modules.eventgrid.sdk.operations'
```

Comment:
Hmm... if the module installation step ran "az eventgrid -h" instead of "az -h" would that have triggered the imports?  If so, perhaps we should beef up our CI script to do that.